mediachain/datastore/utils.py

```diff
@@ -0,0 +1,66 @@
+import base58
+import cbor
+from mediachain.proto import Types_pb2  # pylint: disable=no-name-in-module
+from mediachain.datastore.data_objects import MultihashReference
+
+
+def multihash_ref(ref):
+    if isinstance(ref, MultihashReference):
+        return ref
+
+    try:
```

Comment:
yeah, me too... it seems to be the python way to "ask forgiveness instead of permission", but using `try` like this doesn't feel right to me.  I guess it's a duck-typing thing?